.coveragerc

```diff
@@ -0,0 +1,8 @@
+[run]
+omit = 
+    mbuild/examples/*
+    mbuild/tests/*
+#[report]
+# Regexes for lines to exclude from consideration
+#exclude_lines =
+#    if __name__ == .__main__.:
```

Comment:
Sure, I will include some boiler plate exclusions so that the coverage report is accurate.